cfme/utils/appliance/__init__.py

```diff
@@ -1584,15 +1584,15 @@ def update_guid(self, log_callback=None):
             logger.exception('Exception clearing cached_property "guid"')
         return str(result).rstrip('\n')  # should return UUID from stdout
 
-    def wait_for_ssh(self, timeout=600):
+    def wait_for_ssh(self, timeout=30):
         """Waits for appliance SSH connection to be ready
 
         Args:
             timeout: Number of seconds to wait until timeout (default ``600``)
         """
         wait_for(func=lambda: self.is_ssh_running,
                  message='appliance.is_ssh_running',
-                 delay=5,
+                 delay=1,
```

Comment:
There's a 10s wait within the net_check used by is_ssh_running, so by the time this wait_for wants to try again, the time will have lapsed anyway. Just drop this to a minimum.